koku/api/provider/serializers.py

```diff
@@ -64,7 +64,7 @@ class ProviderBillingSourceSerializer(serializers.ModelSerializer):
 
     uuid = serializers.UUIDField(read_only=True)
     bucket = serializers.CharField(max_length=63, required=True,
-                                   allow_null=False, allow_blank=False)
+                                   allow_null=False, allow_blank=True)
```

Comment:
Do we need this change? Shouldn't we be covered by just having `billing_source` be null?

Reply:
Needed for the Django API view since billing_source is being sent as a parameter